kiva/agg/freetype2/src/tools/docmaker/tohtml.py

```diff
@@ -169,29 +171,29 @@
 # Translate a single line of source to HTML.  This will convert
 # a "<" into "&lt.", ">" into "&gt.", etc.
 def  html_quote( line ):
-    result = string.replace( line, "&", "&amp;" )
-    result = string.replace( result, "<", "&lt;" )
-    result = string.replace( result, ">", "&gt;" )
+    result = line.replace("&", "&amp;" )
+    result = result.replace("<", "&lt;" )
+    result = result.replace(">", "&gt;" )
     return result
 
 
 # same as 'html_quote', but ignores left and right brackets
 def  html_quote0( line ):
-    return string.replace( line, "&", "&amp;" )
+    return line.replace("&", "&amp;" )
 
 
 def  dump_html_code( lines, prefix = "" ):
     # clean the last empty lines
-    l = len( self.lines )
-    while l > 0 and string.strip( self.lines[l - 1] ) == "":
+    l = len( lines )
```

Comment:
I would leave it. The `freetype2` included in kiva is used for its C code and nothing else.